CTP/isolation/src/com/navercorp/cubridqa/isolation/Context.java

```diff
@@ -210,7 +210,7 @@ public boolean isStartMonitor() {
 	}
 
 	public String getTestCaseBranch() {
-		return getProperty("main.testcase.branch_git", "").trim();
+		return getProperty("main.testcase.branch_git", "develop").trim();
```

Comment:
It's better not set default value for main.testcase.branch_git.
Context.java:
this.shouldUpdateTestCase = CommonUtils.convertToBoolean(getProperty("main.testcase.update_yn", "false")) && isNotEmpty(getTestCaseBranch());